npm install --dev gives peerinvalid The package chai does not satisfy its siblings' peerDependencies requirements!

`npm install --dev` I believe is needed to run the tests? If so it cannot succeed:
```
875 error peerinvalid The package chai does not satisfy its siblings' peerDependencies requirements!
876 error System Darwin 15.2.0
877 error command "/usr/local/bin/node" "/usr/local/bin/npm" "install" "--dev"
878 error cwd /work/karma
879 error node -v v0.10.38
880 error npm -v 1.4.28
881 error code EPEERINVALID
882 verbose exit [ 1, true ]
```

Any suggestions?

Thanks!  I updated my node install to 4.2.4 and I got a different -- but still failing -- result:
```
Darwin 15.2.0
npm ERR! argv "/Users/johnjbarton/.nvm/versions/node/v4.2.6/bin/node" "/Users/johnjbarton/.nvm/versions/node/v4.2.6/bin/npm" "install" "--dev"
npm ERR! node v4.2.6
npm ERR! npm  v2.14.12
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package karma-jasmine@0.3.6 does not satisfy its siblings' peerDependencies requirements!
```
Another idea?

(BTW does this mean that in effect node 0.10 is no supported for dev?  Fine by me but just want to know)

Thanks, that worked (I got an error from ghooks but I think it's previously set up). 

Should I send a PR with these instructions in `making-changes`?

`npm test`  fails after bringing up a chrome window that says Karma Starting:
```
Running "test:client" (test) task
Running /work/karma/bin/karma start test/client/karma.conf.js --single-run --no-auto-watch --reporters=dots
22 01 2016 10:38:12.512:INFO [framework.browserify]: bundle built
22 01 2016 10:38:12.521:INFO [karma]: Karma v0.13.19 server started at http://localhost:9876/
22 01 2016 10:38:12.526:INFO [launcher]: Starting browser Chrome
22 01 2016 10:38:14.285:WARN [web-server]: 404: /work/karma/static/karma.js
22 01 2016 10:39:02.513:WARN [launcher]: Chrome have not captured in 50000 ms, killing.
22 01 2016 10:39:02.858:INFO [launcher]: Trying to start Chrome again (1/2).
22 01 2016 10:39:03.979:WARN [web-server]: 404: /work/karma/static/karma.js
22 01 2016 10:39:52.842:WARN [launcher]: Chrome have not captured in 50000 ms, killing.
22 01 2016 10:39:53.150:INFO [launcher]: Trying to start Chrome again (2/2).
22 01 2016 10:39:54.228:WARN [web-server]: 404: /work/karma/static/karma.js
22 01 2016 10:40:43.137:WARN [launcher]: Chrome have not captured in 50000 ms, killing.
22 01 2016 10:40:43.414:ERROR [launcher]: Chrome failed 2 times (timeout). Giving up.
[Error]
Fatal error: Client unit tests failed.
npm ERR! Test failed.  See above for more details.
```

--- a/docs/dev/02-making-changes.md
+++ b/docs/dev/02-making-changes.md
@@ -14,10 +14,19 @@ Here are some tips on how to set up a Karma workspace and how to send a good pul
 
 * Make sure you have a [GitHub account](https://github.com/signup/free).
 * [Fork the repository] on GitHub.
-* Clone your fork.
+* Clone your fork
   ```bash
   $ git clone https://github.com/<your-username>/karma.git
   $ cd karma
+  ```
+* Install for development. Use a recent npm version, ignore peerdep warnings
+  ```bash
+  $ npm install
+  $ rm -rf node_modules/karma
+  $ cd node_modules
+  $ ln -s ../ karma
+  $ cd ../
+  $ grunt browserify
   ```
 
 ## Testing and Building

